app/models/budget.rb

```diff
@@ -30,8 +30,11 @@ class Budget < ActiveRecord::Base
   scope :balloting, -> { where(phase: "balloting") }
   scope :reviewing_ballots, -> { where(phase: "reviewing_ballots") }
   scope :finished, -> { where(phase: "finished") }
+  scope :open, -> { where.not(phase: "finished") }
```

Comment:
I wonder if we should exclude as well 'drafting' budgets, as those should be totally hidden to the public. I know 'drafting' phase wasn't included in this PR (but in one that I created) but just realized/discovered about this filter and that we may not be doing it right.

Reply:
Interesting, it makes sense at a code level, however at the interface level, it's not yet aplicable. As this scope is only used in the admin interface. Where there are only 2 tabs: open and close, which I think fits better with the current definition of open. 

Maybe we can review this when we have a need to extend the budget's admin interface or the user facing interface due to having more budgets in different phases?